src/drives/worksheets/request.rs

```diff
@@ -0,0 +1,97 @@
+// GENERATED CODE
+
+use crate::api_default_imports::*;
+
+resource_api_client!(
+    WorksheetsApiClient,
+    WorksheetsIdApiClient,
+    ResourceIdentity::Worksheets
+);
+
+impl WorksheetsApiClient {
+    post!(
+        doc: "Create new navigation property to worksheets for drives",
+        name: create_worksheets,
+        path: "/worksheets",
+        body: true
+    );
+    get!(
+        doc: "List worksheets",
+        name: list_worksheets,
+        path: "/worksheets"
+    );
+    get!(
+        doc: "Get the number of the resource",
+        name: worksheets,
+        path: "/worksheets/$count"
+    );
+    post!(
+        doc: "Invoke action add",
+        name: add,
+        path: "/worksheets/add",
+        body: true
+    );
+}
+
+impl WorksheetsIdApiClient {
+    get!(
+        doc: "Get Worksheet",
+        name: get_worksheets,
+        path: "/worksheets/{{RID}}"
+    );
+    patch!(
+        doc: "Update worksheet",
+        name: update_worksheets,
+        path: "/worksheets/{{RID}}",
+        body: true
+    );
+    delete!(
+        doc: "Worksheet: delete",
+        name: delete_worksheets,
+        path: "/worksheets/{{RID}}"
+    );
+    get!(
+        doc: "Invoke function cell",
+        name: cell,
+        path: "/worksheets/{{RID}}/cell(row={{id}},column={{id2}})",
+        params: row, column
+    );
+    delete!(
+        doc: "Delete navigation property protection for drives",
+        name: delete_protection,
+        path: "/worksheets/{{RID}}/protection"
+    );
+    get!(
+        doc: "Get WorksheetProtection",
+        name: get_protection,
+        path: "/worksheets/{{RID}}/protection"
+    );
+    patch!(
+        doc: "Update the navigation property protection in drives",
+        name: update_protection,
+        path: "/worksheets/{{RID}}/protection",
+        body: true
+    );
+    post!(
+        doc: "Invoke action protect",
+        name: protect,
+        path: "/worksheets/{{RID}}/protection/protect",
+        body: true
+    );
+    post!(
+        doc: "Invoke action unprotect",
+        name: unprotect,
+        path: "/worksheets/{{RID}}/protection/unprotect"
+    );
+    get!(
+        doc: "Invoke function range",
+        name: workbook_worksheet,
+        path: "/worksheets/{{RID}}/range()"
+    );
+    get!(
+        doc: "Invoke function range",
+        name: workbook_worksheet,
+        path: "/worksheets/{{RID}}/range(address='{{id}}')",
+        params: address
+    );
```

Comment:
So I think I fail to mention in the walk through the other way to handle methods with the same names. This one is pretty easy to change by just updating the settings again in codegen.

In this case the method names need to be changed instead of generating a separate client since its only two methods and its not really part of a separate API.

This would be added at the end of the match here: https://github.com/sreeise/graph-rs-sdk/blob/c1301045b52c39d132851b7e4379896cb3717b37/graph-codegen/src/settings/method_macro_modifier.rs#L732C5-L732C5

```rust
ResourceIdentity::Workbooks => vec![
	MethodMacroModifier::fn_name_and_path(
		"workbook_worksheets", "/worksheets/{{RID}}/range()",
		GeneratedMacroType::FnName("get_range_object")
	),
	MethodMacroModifier::fn_name_and_path(
		"workbook_worksheets", "/worksheets/{{RID}}/range(address='{{id}}')",
		GeneratedMacroType::FnName("get_range_object_by_address")
	)
],
```

The fn_name_and_path function takes three arguments:
1. The method name that we want to change.
2. The path of the api call
3. The type and value that we want to update. In this case we just want to update the method name.

And the naming I chose, get_range_object and get_range_object_by_address, is just based off general info in the docs here: https://learn.microsoft.com/en-us/graph/api/worksheet-range?view=graph-rest-1.0&tabs=http

That way we can remove ResourceIdentity::WorksheetsUsedRange and also remove the API client generated for it.

Reply:
Done, remove used range client and renamed 4 methods